src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -3162,23 +3162,7 @@ public function isDirty($attributes = null)
      */
     public function isPristine($attributes = null)
     {
-        $dirty = $this->getDirty();
-
-        if (is_null($attributes)) {
-            return count($dirty) === 0;
-        }
-
-        if (! is_array($attributes)) {
-            $attributes = func_get_args();
-        }
-
-        foreach ($attributes as $attribute) {
-            if (array_key_exists($attribute, $dirty)) {
-                return false;
-            }
-        }
-
-        return true;
+        return ! $this->isDirty(...func_get_args());
```

Comment:
Yeah, with the `call_user_func_array`. Should I change it or make a new PR to 5.3?